examples/Console/TestPrometheusExporter.cs

```diff
@@ -48,15 +48,17 @@ internal static object Run(int port)
               - targets: ["localhost:9464"]
         */
 
+        Action<OpenTelemetry.Exporter.Prometheus.HttpListener.PrometheusHttpListenerOptions> configureListenerOptions = listenerOptions =>
+                    listenerOptions.Prefixes = new string[] { $"http://localhost:{port}/" };
         using var meterProvider = Sdk.CreateMeterProviderBuilder()
             .AddMeter(MyMeter.Name)
             .AddMeter(MyMeter2.Name)
-            .AddPrometheusExporter(options =>
+            .AddPrometheusHttpListener(
+                exporterOptions =>
             {
-                options.StartHttpListener = true;
-                options.HttpListenerPrefixes = new string[] { $"http://localhost:{port}/" };
-                options.ScrapeResponseCacheDurationMilliseconds = 0;
-            })
+                exporterOptions.ScrapeResponseCacheDurationMilliseconds = 0;
+            },
+                configureListenerOptions: configureListenerOptions)
```

Comment:
nit: It might read a bit better like...

```csharp
        using var meterProvider = Sdk.CreateMeterProviderBuilder()
            .AddMeter(MyMeter.Name)
            .AddMeter(MyMeter2.Name)
            .AddPrometheusHttpListener(
                exporterOptions => exporterOptions.ScrapeResponseCacheDurationMilliseconds = 0,
                listenerOptions => listenerOptions.Prefixes = new string[] { $"http://localhost:{port}/" })
```

Reply:
Fixed here and also in other places with similar patterns. Thanks!